rocAL/rocAL/source/video_properties.cpp

```diff
@@ -133,6 +133,7 @@ void get_video_properties_from_txt_file(VideoProperties &video_props, const char
                 THROW("Variable frame rate videos cannot be processed")
             video_props.frame_rate = video_frame_rate;
             video_count++;
+            props.clear();
```

Comment:
it is better to have a structure for props instead of defining it as a vector of unsigned int.

Reply:
@Indumathi31 : Can you please address this?

Reply:
> @Indumathi31 : I believe the sequence reader tests were succeeding before. What changes made it output blank images? We need to get this fixed so we can merge it

@rrawther @paveltc Fix patch is still in progress. I am comparing the differences between previous working commit with current version. I will let you know the status sooner.

Reply:
@rrawther We have added this support and I will update in the same PR

Reply:
> @Indumathi31 : Can you please address this?

Have added the changes.

Reply:
> it is better to have a structure for props instead of defining it as a vector of unsigned int.

Have added this change